ansible/roles/data-products-deploy/tasks/main.yml

```diff
@@ -97,14 +97,14 @@
   tags:
     - update-jobmanager-config
 
-- name: Update user redis cache
-  cron: name="{{env}}-{{ item.key }}" minute={{ item.value.minute }} hour={{ item.value.hour }}  job="cd {{ analytics.home }} && adhoc-jobs-1.0/scripts/RedisUserDataIndexer.sh"
-  with_dict: "{{ update_user_redis_cache }}"
-  tags:
-    - cronjobs
-
-- name: Create content-model-snapshot-druid-indexer cron job
-  cron: name="{{env}}-{{ item.key }}" minute={{ item.value.minute }} hour={{ item.value.hour }}  job="cd {{ analytics.home }} && adhoc-jobs-1.0/scripts/DruidContentIndexer.sh"
-  with_dict: "{{ index_content_model_druid }}"
-  tags:
-    - cronjobs
+#- name: Update user redis cache
+#  cron: name="{{env}}-{{ item.key }}" minute={{ item.value.minute }} hour={{ item.value.hour }}  job="cd {{ analytics.home }} && adhoc-jobs-1.0/scripts/RedisUserDataIndexer.sh"
```

Comment:
Why do we need commented line? If this is temporary, its fine, else please remove